example/ndpiReader.c

```diff
@@ -523,6 +523,11 @@ static void help(u_int long_help) {
 #endif
 
   if(long_help) {
+    printf("\n\nSize of nDPI Flow structure: %u\n"
+           "Sizeof of nDPI Flow protocol union: %zu\n",
+           ndpi_detection_get_sizeof_ndpi_flow_struct(),
+           sizeof(((struct ndpi_flow_struct *)0)->protos));
+
```

Comment:
Useful. I think is will be more useful to have the size of the entire `struct ndpi_flow_struct`. Or both?

Reply:
Probably better to keep both, because of the two `_Static_assert`s in `ndpi_typedefs.h`.